updatecli/updatecli.d/patch.yaml

```diff
@@ -0,0 +1,21 @@
+name: Update chart patch version
+
+sources:
+  chart:
+    name: Load chart version
+    kind: yaml
+    spec:
+      file: "file://charts/{{ requiredEnv .chart  }}/Chart.yaml"
+      key: "version"
+
+
+targets:
+  chart-patch-version-update:
+    name: Bump chart patch version
+    kind: yaml
+    sourceid: chart
+    transformers:
+      - semverinc: "patch"
+    spec:
+      file: "file://charts/{{ requiredEnv .chart  }}/Chart.yaml"
+      key: "version"
```

Comment:
don't we have to bump also the `appVersion`?

There are also a ton of rancher specific variables that hold a reference to application version. These are the painful ones to update.
For example, take a look at this commit: https://github.com/kubewarden/helm-charts/commit/5d351b0d4ddb84b0789f19ead23d5d7a64cd54b1

Reply:
> don't we have to bump also the `appVersion`?

Supposedly not, [by the rfc](https://github.com/kubewarden/rfc/blob/main/rfc/0007-kubewarden-versioning.md#a-patch-is-done-to-kubewarden-controller).

Reply:
@jvanz I think we have to update this updatecli "configuration" to keep the `version` and the `catalog.cattle.io/upstream-version` values in sync, see what we did manually inside of https://github.com/kubewarden/helm-charts/commit/5d351b0d4ddb84b0789f19ead23d5d7a64cd54b1